tests/suites/1_stateful/00_copy/00_0000_copy_from_stage.sh

```diff
@@ -72,3 +72,39 @@ curl -s -u root: -XPOST "http://localhost:${QUERY_HTTP_HANDLER_PORT}/v1/query" -
 echo "drop table ontime200" | $MYSQL_CLIENT_CONNECT
 echo "drop stage if exists named_external_stage" | $MYSQL_CLIENT_CONNECT
 echo "drop stage if exists s1" | $MYSQL_CLIENT_CONNECT
+
+
+## Test
+echo "drop stage if exists s1" | $MYSQL_CLIENT_CONNECT
+echo "drop stage if exists named_external_stage" | $MYSQL_CLIENT_CONNECT
+
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv s3://testbucket/admin/stage/s1/ontime_200.csv >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.gz s3://testbucket/admin/stage/s1/ontime_200.csv.gz >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.zst s3://testbucket/admin/stage/s1/ontime_200.csv.zst >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.bz2 s3://testbucket/admin/stage/s1/ontime_200.csv.bz2 >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.parquet s3://testbucket/admin/stage/s1/ontime_200.parquet >/dev/null 2>&1
+
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv s3://testbucket/admin/tempdata/ontime_200.csv >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.gz s3://testbucket/admin/tempdata/ontime_200.csv.gz >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.zst s3://testbucket/admin/tempdata/ontime_200.csv.zst >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.csv.bz2 s3://testbucket/admin/tempdata/ontime_200.csv.bz2 >/dev/null 2>&1
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.parquet s3://testbucket/admin/tempdata/ontime_200.parquet >/dev/null 2>&1
+
+
+## Copy from internal stage
+echo "CREATE STAGE s1;" | $MYSQL_CLIENT_CONNECT
+## Remove internal stage file
+echo "remove @s1/ontime_200.csv.gz" | $MYSQL_CLIENT_CONNECT
+echo "list @s1 PATTERN = 'ontime.*'" | $MYSQL_CLIENT_CONNECT
+## Remove internal stage file with pattern
+echo "remove @s1  PATTERN = 'ontime.*'" | $MYSQL_CLIENT_CONNECT
+
+## Copy from named external stage
+echo "CREATE STAGE named_external_stage url = 's3://testbucket/admin/tempdata/' credentials=(aws_key_id='minioadmin' aws_secret_key='minioadmin');" | $MYSQL_CLIENT_CONNECT
+echo "list @named_external_stage PATTERN = 'ontime.*'" | $MYSQL_CLIENT_CONNECT
```

Comment:
```suggestion
echo "list @named_external_stage PATTERN = 'ontime.*'" | $MYSQL_CLIENT_CONNECT | sort 
```

Reply:
get! thank you for help.

Reply:
But I did not find a similar `chmod +x 00_0000_copy_from_s3_location.sh ?

Reply:
It's to make `00_0000_copy_from_s3_location.sh` can be executed.

You need to execute it locally and commit to git.

See  https://linuxtect.com/what-is-chmod-x-command-in-linux/